File: src/B_5596/B_5596.java
package B_5596;

import java.io.*;
import java.util.*;

public class B_5596 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] a1 = br.readLine().split(" ");
        int S = 0;
        for (String s :
                a1) {
            S += Integer.parseInt(s);
        }
        String[] a2 = br.readLine().split(" ");
        int T = 0;
        for (String s :
                a2) {
            T += Integer.parseInt(s);
        }
        System.out.println(Math.max(S,T));
    }
}

File: src/B_5596/B_5596.kt
package B_5596

import java.io.*
import kotlin.math.max

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val a1: Array<String> = readLine().split(" ").toTypedArray()
    var S: Int = 0
    for(i in a1){
        S += i.toInt()
    }
    val a2: Array<String> = readLine().split(" ").toTypedArray()
    var T:Int = 0
    for(i in a2){
        T += i.toInt()
    }
    print(max(S,T))
}